l1-contracts/contracts/interop/InteropHandler.sol

```diff
@@ -315,6 +315,10 @@ contract InteropHandler is IInteropHandler, ReentrancyGuard {
 
         bundleStatus[_bundleHash] = BundleStatus.Verified;
 
+        /// We send the fact of verification to L1 so that the GWAssetTracker can process the chainBalance changes.
+        // slither-disable-next-line reentrancy-no-eth,unused-return
+        L2_TO_L1_MESSENGER_SYSTEM_CONTRACT.sendToL1(bytes.concat(this.verifyBundle.selector, _bundleHash));
```

Comment:
is there anything that enforces that it happens on GW? Otherwise a chain might mint internally tokens that it will not have access to

Reply:
What happens on GW? The logs are sent to GWAT, so the _handleInteropHandlerReceiveMessage processes it

Reply:
If a bundle is executed and the chain mints funds to itself while settling on L1 its internal `chainBalance` will increase, while its chainBalance on L1 wont. 

So in any place that we send this message from L2 we need to ensure that the settlement layer of the chain is GW

Reply:
https://github.com/matter-labs/era-contracts/pull/1703/commits/ca7ac52629b718fa5d436133e6681dc8a324fd8a